modules/appeals_api/spec/lib/decision_review_report_spec.rb

```diff
@@ -5,6 +5,25 @@
 
 describe AppealsApi::DecisionReviewReport do
   # rubocop:disable Layout/FirstHashElementIndentation
+  shared_examples 'stuck appeals' do |opts|
+    it 'retrieves records stuck in incomplete statuses' do
+      stuck_appeal_no_updates = stuck_appeal_with_updates = unstuck_appeal = nil
```

Comment:
I like the fact that this shortens the line count... But I think I would lean towards more readability (and more lines) in this case...